### DISCUSSION TITLE
How to add title for listing images, and alternative text for listing placeholder image?

### DISCUSSION THREAD

**Discussion by maelle** [quarto-dev/quarto-cli#6987] [Q&A]
### Description

I have two closely related questions about images for [website listings](https://quarto.org/docs/websites/website-listings.html)

- How could one add the title for an image, that is to say the text available on hover? I was thinking it'd be a good place to add credit (to the image creator).
- When using an image [placeholder](https://quarto.org/docs/websites/website-listings.html#default-1), can one add an alternative text?

Thank you. :pray:

**Comment by dragonstyle** [comment]
> How could one add the title for an image, that is to say the text available on hover? I was thinking it'd be a good place to add credit (to the image creator).

Right now we don't allow you to set this, but your idea is a good one! 

> When using an image [placeholder](https://quarto.org/docs/websites/website-listings.html#default-1), can one add an alternative text?

You can't right now, though this too would make sense (e.g. `image-placeholder-alt` or the like).

Sorry we don't currently support this but these would both be excellent enhancements! I'll open enhancements to track.

**Reply by mcanouil** [reply]
Closing in favour of:
- #7027
- #7028

**Reply by maelle** [reply]
Thanks both!! I'll subscribe :smile_cat:
